packages/perseus/src/widgets/radio/multiple-choice-option.new.tsx

```diff
@@ -0,0 +1,53 @@
+import React, {useRef} from "react";
+
+import Indicator from "./choice-indicator.new";
+import styles from "./choice.module.css";
+
+export interface IndicatorProps {
+    checked: boolean;
+    children: React.ReactNode | React.ReactNode[];
+    indicatorContent: string;
+    isMultiSelect: boolean;
+    showCorrectness?: "correct" | "wrong";
+    updateChecked: (isChecked: boolean) => void;
+}
+
+// TODO: Change name of this file to choice.new.tsx (replacing existing one)
+//           and its corresponding test and Storybook files.
+
+const Choice = (props: IndicatorProps) => {
+    const showCorrectness = props.showCorrectness;
+    const buttonRef = useRef<HTMLButtonElement>(null);
+    const indicatorShape = props.isMultiSelect ? "square" : "circle";
+    const clickHandler = showCorrectness
+        ? undefined
+        : () => {
+              buttonRef.current?.click();
+          };
+    const classes = [styles.choice]
+        .concat(showCorrectness ? [styles["is-" + showCorrectness]] : [])
+        .join(" ");
+    return (
+        /*
+            A click handler is added to the <li> element to accommodate mouse users.
```

Comment:
Thank you for this helpful comment, as I'm sure this will help us one day in the future when we've forgotten the context.